scripts/gulp/src/plugins/util/getShorthandInfo.ts

```diff
@@ -1,6 +1,7 @@
 import * as Babel from '@babel/core';
 import { NodePath } from '@babel/traverse';
-import * as t from '@babel/types';
+import type * as t from '@babel/types';
+import { assertStringLiteral, isIdentifier } from '@babel/types';
```

Comment:
`t.<method>` cant be used as it will throw  https://github.com/microsoft/TypeScript/issues/47945 all the time. thus using direct functions is the only way with `asserts`